### DISCUSSION TITLE
逆向破解学校在线考试客户端

### DISCUSSION THREAD

**Discussion by giscus** [zu1k/blog#57] [Comments]
# 逆向破解学校在线考试客户端

A boy dreaming of traveling around the world.<br>This blog is used to record my study, thinking and life.

https://zu1k.com/posts/security/reverse/crack-sdu-online-exam-software/#giscus

**Comment by NanFengXiangWan** [comment]
请问现在还能用么

**Reply by zu1k** [reply]
现在不用这个了，这个是疫情期间在家线上考试时候用的

**Comment by yuantuo666** [comment]
文章最后的 asset_link 寄了 :shipit:

**Reply by zu1k** [reply]
其实并没有

**Comment by thirteenkai** [comment]
广开的考试客户端如何破解呢
